src/core/reopt_inputs.jl

```diff
@@ -1208,6 +1230,20 @@ function fillin_techs_by_exportbin(techs_by_exportbin::Dict, tech::AbstractTech,
     return nothing
 end
 
+function fillin_storage_by_exportbin(s::AbstractScenario, storage_by_exportbin::Dict, b::String)
```

Comment:
lower priority, but noting: I'd like to modify this function's args to align with fillin_techs_by_exportbin, i.e. passing in storage struct instead of whole scenario